Automattic/woocommerce-payments
# Adjust WooPay login modal viewport height unit

Fixes 1689-gh-Automattic/woopay

#### Changes proposed in this Pull Request

Changes the viewport height unit for the WooPay login modal from `vh` to `dvh`.









#### Testing instructions

1. Check out this branch and 1966-gh-Automattic/woopay
2. On a mobile device, BrowserStack, or iOS Simulator, tap the Buy with WooPay button while the browser URL bar or toolbar is still on the screen. This is easiest from the checkout page where the WooPay button is at the top of the page.
3. Check that the Continue button is fixed to the bottom of the screen and is fully accessible with or without the browser URL or toolbar on screen and that no pinch/zooming is necessary

-------------------


**Post merge**
